Cargo.lock

```diff
@@ -313,9 +313,9 @@ checksum = "7de8ce5e0f9f8d88245311066a578d72b7af3e7088f32783804676302df237e4"
 
 [[package]]
 name = "async-stream"
-version = "0.3.4"
+version = "0.3.5"
```

Comment:
Should we be upgrading dependencies so close to release?

Reply:
I'll restore the Cargo.lock, I guess.